src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,50 @@
 import React from 'react';
-
+import { Movie } from '../../types/Movie';
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+interface MoviesCardProps {
+  movie: Movie;
+}
+
+export const MovieCard: React.FC<MoviesCardProps> = ({ movie }) => (
+  <div className="card" data-cy="Movie">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          data-cy="MovieImage"
+          src={movie.imgUrl}
+          alt="Film logo"
+        />
+      </figure>
+    </div>
+
+    <div className="card-content">
+      <div className="media">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img src="images/imdb-logo.jpeg" alt="imdb" />
+          </figure>
+        </div>
+
+        <div className="media-content">
+          <p className="title is-8" data-cy="MovieTitle">
+            {movie.title}
+          </p>
+        </div>
+      </div>
+
+      <div className="content">
+        <p data-cy="MovieDescription">
+          {movie.description}
+        </p>
+
+        <a
+          href={movie.imdbUrl}
+          data-cy="MovieLink"
+        >
```

Comment:
wrap if you have more than 2 params or length > 80
```suggestion
        <a href={movie.imdbUrl} data-cy="MovieLink">
```